src/tracer.js

```diff
@@ -202,29 +202,40 @@ function getTrimmedMetadata(eventMetadata, isRunner) {
 function getTrimmedTrace(traceSize, jsTrace) {
     let currentTraceSize = traceSize;
     const trimmedTrace = Object.assign({}, jsTrace);
-    trimmedTrace.events = jsTrace.events.sort(event => (['runner', 'trigger'].includes(event.origin) ? -1 : 1));
+    let totalTrimmedExceptions = 0;
```

Comment:
general note: try to keep it to 1 PR per 1 feature/fix (meaning this should've been a separate PR). Also don't forget to fix the PR title / commit message to take this change into account.

Reply:
you're right, I'll export the new STRONG_ID_KEYS params commit to a new PR.